0-ad-history-mod-package/simulation/templates/mixins/encyclopedia/encyclopedia_athens_walls

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Entity>
+  <Identity>
+    <History>
+        The idea to uniformly build walls was probably first proposed by the politician Themistocles after Athens was completely demolited during the second Persian invasion. While the walls around the city were constructed right away, the famous Long Walls would not be until 20 years later, around 460 BC.
+        The two Long Walls each measuring 6 kilometers in length, stood parallel, 200 meters apart, protecting the passage to Athens' port, Piraeus. Another wall further to the east was also constructed towards another smaller port called Phaleron, forming a triangle enclosed by the walls and the shore that could be used for agriculture. The entire Athenian wall system was over 25 kilometers long, around 3 meters wide, and 9 meters high, and became a majestic symbol for Athens' power and dominance.
+        The walls formed a key point in their war strategy; Athens could avoid fighting enemies on land and instead rely on their powerful navy to engage in combat with them. And, in case of a siege, the city had coastal access within its walls, which meant all necessary food could be imported from the sea. 
```

Comment:
```suggestion
        The walls formed a key point in Athens's war strategy; They could avoid fighting enemies on land and instead rely on their powerful navy to engage in combat with them. And, in case of a siege, the city had coastal access in its walls, which meant all necessary food could be brought from the sea. 
```